yarn.lock

```diff
@@ -143,6 +143,11 @@ find-up@^3.0.0:
   dependencies:
     locate-path "^3.0.0"
 
+fs@^0.0.1-security:
+  version "0.0.1-security"
+  resolved "https://registry.yarnpkg.com/fs/-/fs-0.0.1-security.tgz#8a7bd37186b6dddf3813f23858b57ecaaf5e41d4"
+  integrity sha1-invTcYa23d84E/I4WLV+yq9eQdQ=
+
```

Comment:
These should be also removed after the dependency is removed.

Reply:
resolved